package.json

```diff
@@ -46,6 +46,7 @@
     "@storybook/addon-actions": "^6.3.6",
     "@storybook/addon-backgrounds": "^6.3.6",
     "@storybook/addon-docs": "^6.3.6",
+    "@storybook/addon-essentials": "^6.4.13",
     "@storybook/addon-knobs": "^6.3.0",
```

Comment:
I haven't updated all of Storybook yet. This is so that I can still use Storybook while migrating fro knobs to controls. Once I have removed all knobs,I can remove `@storybook/addon-knobs` and  I will update to 6.4.13 all around.